examples/example_plotly_fixed_axis_moving_in_range.py

```diff
@@ -9,13 +9,14 @@
 
 #################################
 projection_n_dimensions = 3
-nfs = NeoForceScheme(metric="euclidean", verbose=True, learning_rate0=0.5,
-                     decay=0.95, max_it=100, cuda=False)
-starting_projection_mode = ProjectionMode.RANDOM
+nfs = NeoForceScheme(metric="euclidean", verbose=True, learning_rate0=0.2,
+                     decay=0.95, max_it=300, cuda=False)
+starting_projection_mode = ProjectionMode.TSNE
 fix_column_to_z_projection_axis = -1
 drop_columns_from_dataset = [-1]
 scaler = (0, 1)
 z_axis_moving_range = (-0.2, 0.2)
+confidence_interval = 0.9
```

Comment:
I imagine there is no difference between confidence_interval=0.9 and 0.99, correct? Just wondering..

Reply:
since the z scores (and the shapes of the gaussion function) for 0.9 and 0.99 are different, I think they should be different. for example, if we set the moving range to be (-1, 1), 0.9 means 90% of the area under the line of the gaussian function is between -1 and 1, while 0.99 means 99% of the area under the line is between -1 and 1. More specifically, 0.99 means the peak of the function figure (i.e x = 0) should be more sharp, if my calculation is correct.

Reply:
This is a nice explanation. In the previous comment, I said something to include in the :param comment. Maybe this could be it :)